src/MassTransit.Abstractions/IMetricsContext.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace MassTransit
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Provides access to tags added to metrics.
+    /// </summary>
+    public interface IMetricsContext
```

Comment:
`MetricsContext`?
I assume you want to expose this outside MT lib?

Reply:
Yes, should be without the `I`.